### PR TITLE
Deprioritize SDK content

### LINKED ISSUE #1196: Deprioritize SDK content
We have been requested to deprioritize the SDK content for now. This issue is to:
- Move the "Use MetaMask SDK" category lower in the "How to" category.
- Remove the "SDK" label from the top navigation.
- Until the quickstarts are removed entirely in #1197, remove the SDK options from the quickstarts.
- Remove the SDK as a "third-party library" option for detecting multiple wallets. This is in conjunction with [this issue](#1197) to emphasize and improve the EIP-6963 content.
  - [Detect multiple wallets](https://docs.metamask.io/wallet/how-to/detect-wallet/multiple-wallets/#use-third-party-libraries)
  - [Wallet interoperability](https://docs.metamask.io/wallet/concepts/wallet-interoperabilty/#third-party-library-support)

### PR DESCRIPTION
**Pull request by alexandratran**
This PR:

- Removes the "SDK" label from the top navigation.
- Moves "Use MetaMask SDK" category lower in the "How to" category.
- Removes the SDK as a third-party library option for detecting multiple wallets (there's still a section in the Wallet Interoperability concepts page explaining the SDK EIP-6963 functionality).
- Removes the SDK options from the quickstarts.

Fixes #1196

### CODE REVIEW

File: wallet/concepts/wallet-interoperabilty.md [hunk 1]
```diff
@@ -27,23 +27,26 @@ for more information.
...
 - [MIPD Store](https://github.com/wevm/mipd)
-
 - [RainbowKit](https://www.rainbowkit.com/)
 
-### MetaMask SDK
+### MIPD Store
+
+The [MIPD Store](https://github.com/wevm/mipd) stores the wallet providers and enables you to
+subscribe to the store and retrieve the providers.
+Unlike Wagmi and Web3-Onboard, which are libraries that provide components and connectors for
```
Comment (joaniefromtheblock) [inline wallet/concepts/wallet-interoperabilty.md:39]: Should link to these libraries?

File: wallet/concepts/wallet-interoperabilty.md [hunk 2]
```diff
@@ -27,23 +27,26 @@ for more information.
...
+multiple wallets and depend on MetaMask SDK for integration, the MIPD Store is a utility library
+that makes it easier to work with EIP-6963 and supports TypeScript types.
+
+## MetaMask SDK support
+
+:::note
+MetaMask SDK does not support connecting to non-MetaMask wallets via EIP-6963.
+If you intend to support discovery of other wallets, we recommend using other methods of adding
+EIP-6963 support such as Wagmi 2+.
```
Comment (joaniefromtheblock) [inline wallet/concepts/wallet-interoperabilty.md:48]: Maybe a link to this for the ease of the user

**Review by joaniefromtheblock — APPROVED** [review]
LGTM

**Review by subhamkumarr — APPROVED** [review]

**Comment by subhamkumarr** [comment]
LGTM

**Review by Montoya — APPROVED** [review]

**Pull request merged**